src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -23,7 +23,7 @@ module sonic-portchannel {
 	description "PORTCHANNEL yang Module for SONiC OS";
 
 	revision 2019-07-01 {
-		description "First Revision";
+		description "Increased range for min_links";
```

Comment:
Yang model subgroup meeting 4/8: 
Please confirm if revision needs to be changed. (cc: @praveen-li)

Reply:
For each update to the yang, it is better to maintain a corresponding revision with the description of the change.